bin/genpng

```diff
@@ -271,14 +272,28 @@ sub gen_png($$$@)
 		or die("ERROR: cannot allocate overview image!\n");
 
 	# Define colors
-	$col_plain_back	= $overview->colorAllocate(0xff, 0xff, 0xff);
-	$col_plain_text	= $overview->colorAllocate(0xaa, 0xaa, 0xaa);
-	$col_cov_back	= $overview->colorAllocate(0xaa, 0xa7, 0xef);
-	$col_cov_text	= $overview->colorAllocate(0x5d, 0x5d, 0xea);
-	$col_nocov_back = $overview->colorAllocate(0xff, 0x00, 0x00);
-	$col_nocov_text = $overview->colorAllocate(0xaa, 0x00, 0x00);
-	$col_hi_back = $overview->colorAllocate(0x00, 0xff, 0x00);
-	$col_hi_text = $overview->colorAllocate(0x00, 0xaa, 0x00);
+	# overview->colorAllocate(red, green, blue)
+	if ($dark_mode) {
+	  # just reverse foregrond and background
+	  #  there is probably a better color scheme than this.
+	  $col_plain_text = $overview->colorAllocate(0xff, 0xff, 0xff);
+	  $col_plain_back = $overview->colorAllocate(0xaa, 0xaa, 0xaa);
```

Comment:
I like the color scheme with one proposed change: use light-gray (`#aaaaaa`) for `$col_plain_text` and black (`#000000`) for `$col_plain_back`. This will make it look even more like a miniature version of the actual color scheme produced by genhtml